_posts/2023-03-06-xenforum.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: post
+title: "[CVE-2023-24763] Multiple improper neutralizations of an SQL parameters in XenForum module for PrestaShop"
+categories: modules
+author:
+- 202-ecommerce.com
+- TouchWeb.fr
+- Friends-Of-Presta.org
+meta: "CVE,PrestaShop,xenforum"
+severity: "high (8.8)"
+---
+
+In the module "Xen Forum" (xenforum) edited by App1pro, an authenticated user can perform SQL injection in affected versions. 2.13.0 fixed vulnerabilities.
+
+## Summary
+
+* **CVE ID**: [CVE-2023-24763](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-24763)
+* **Published at**: 2023-03-06
+* **Advisory source**: Friends-of-Presta.org
+* **Vendor**: PrestaShop
+* **Product**: xenforum
+* **Impacted release**: < 2.13.0
+* **Product author**: App1pro
+* **Weakness**: [CWE-89](https://cwe.mitre.org/data/definitions/89.html)
+* **Severity**: high (8.8)
+
+## Description
+
+In Xen Forum module for PrestaShop up to 2.13.0, multiple sensitives SQL calls in class `XenForumAttachment::getListAttachments()` (or `XenForumCat`, `XenForumPost`, ...) can be executed with a trivial http call and exploited to forge a bind SQL injection.
+
+## CVSS base metrics
+
+* **Attack vector**: network
+* **Attack complexity**: low
+* **Privilege required**: low
+* **User interaction**: none
+* **Scope**: unchanged
+* **Confidentiality**: high
+* **Integrity**: high
+* **Availability**: high
+
+**Vector string**: CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H
+
+## Possible malicious usage
+
+* Technical and personal data leaks
+* Obtain admin access
+* Remove all data of the linked PrestaShop
+* Display sensitives tables to front-office to unlock potential admin's ajax scripts of modules protected by token on the ecosystem
+
+## Proof of concept
+
+```bash
+curl -v --cookie-jar cookie.txt 'https://domain.tld/authentification?submitLogin=1&emailXXXX&password=YYY && \
+curl -v --cookie cookie.txt 'http://domain.tld/index.php?fc=module&module=xenforum&id=194&controller=editpost&edit_post=1&attachments[]=3%29%3BSELECT%20SLEEP%2825%29%23'
+```
+
+## Patch from 2.7.6
```

Comment:
I am not sure can you check the version ?

Reply:
Thanks